src/vnsw/agent/oper/interface.h

```diff
@@ -171,6 +175,10 @@ class Interface : AgentRefCount<Interface>, public AgentOperDBEntry {
     Transport transport_;
     AgentQosConfigConstRef qos_config_;
 
+    #ifdef _WIN32
```

Comment:
"Even when preprocessor directives are within the body of indented code, the directives should start at the beginning of the line."